config.yml

```diff
@@ -731,6 +731,8 @@ nodes:
         type: node?
       - name: end_keyword_loc
         type: location
+      - name: name
+        type: string
```

Comment:
It could likely use the constant pool, but it seems constants in general don't use the pool yet.